_commands/basics/man.md

```diff
@@ -3,22 +3,31 @@
 
 man
 --
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+`man` provides *manual pages* for terminal commands
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ man git
+q // to quit
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
 
-#### Example command
-
-##### Break it down
-
-#### Example command
+#### `man -f <command>` shows a list of related commands that match the <command> keyword
+~~~ bash
+$ man -f git
+perlgit(1)               - Detailed information about git and the Perl repository
+git(1)                   - the stupid content tracker
+git-add(1)               - Add file contents to the index
+git-am(1)                - Apply a series of patches from a mailbox
+git-annotate(1)          - Annotate file lines with commit information
+git-apply(1)             - Apply a patch to files and/or to the index
+git-archimport(1)        - Import an Arch repository into Git
+git-archive(1)           - Create an archive of files from a named tree
+git-bisect(1)            - Use binary search to find the commit that introduced a bug
+...
+~~~
 

```

Comment:
At the bottom of some reference pages, there are links to other relevant commands. Linking to the `help` and `apropros` commands could be useful here.